lib/tapioca/autoload_tracker.rb

```diff
@@ -37,8 +39,8 @@ def with_disabled_exits(&block)
         original_exit = Kernel.instance_method(:exit)
 
         begin
-          Kernel.define_method(:abort) {}
-          Kernel.define_method(:exit) {}
+          Kernel.define_method(:abort, NOOP_METHOD)
+          Kernel.define_method(:exit, NOOP_METHOD)
```

Comment:
Should this be a list defined as a constant somewhere so we iterate over the list for no-op each method?